src/test/java/org/tarantool/AsyncClientOperationsIT.java

```diff
@@ -0,0 +1,119 @@
+package org.tarantool;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Tests for asynchronous operations provided by {@link TarantoolClientImpl} class.
+ */
+public class AsyncClientOperationsIT extends AbstractTarantoolConnectorIT {
```

Comment:
It is possible to test asynchronous operations via AbstractTarantoolOpsIT, by getOps() that wait result of an operation. I don't have objections against existing tests, just note it.

Reply:
OK